TimeLog.ReportingAPI.SDK/Connected Services/TimeLog.ReportingAPI.SDK/Reference.cs

```diff
@@ -1068,11 +1102,6 @@ public virtual System.Threading.Tasks.Task OpenAsync()
             return System.Threading.Tasks.Task.Factory.FromAsync(((System.ServiceModel.ICommunicationObject)(this)).BeginOpen(null, null), new System.Action<System.IAsyncResult>(((System.ServiceModel.ICommunicationObject)(this)).EndOpen));
         }
         
-        public virtual System.Threading.Tasks.Task CloseAsync()
-        {
```

Comment:
why are we removing this snippet?

Reply:
The code was auto-generated by the dotnet-svcutil tool. I believe there have been changes to the metadata generated after the .NET 6 SDK upgrade, which includes updates to the tool.